src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -83,14 +85,14 @@ TEST_F(StopPlayTest, test_stop_play_friendly_half_corner_robots_close_together)
 {
     BallState ball_state(Point(-4, -2.5), Vector(0, 0));
     auto friendly_robots = TestUtil::createStationaryRobotStatesWithId(
-        {Point(-3, -2.5), Point(-4, -2), Point(-2, -2.5), Point(-3, -2), Point(-3.5, -2),
+        {Point(-2, -2.5), Point(-4, -2), Point(-2, -2.5), Point(-3, -2), Point(-3.5, -2),
```

Comment:
is this required for the test to pass?

Reply:
Just disabled the test given our discussion last weekend